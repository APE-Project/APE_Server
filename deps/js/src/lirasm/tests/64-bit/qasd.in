one = immq 1
d = immd 123.45
q = dasq d
q2 = addq q one     ; do some intermediate stuff just to complicate things
q3 = subq q2 one
d2 = qasd q3
retd d2
