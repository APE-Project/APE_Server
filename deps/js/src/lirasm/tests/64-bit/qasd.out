Output is: 123.45
